src/index.scss

```diff
@@ -268,37 +258,44 @@ mobile has scaling widths and font sizes
   }
 
   .hero-img{
-    width: 90%;
+    /* slightly smaller hero img for shorter screens */
+    min-width: 80%;
+    width: unquote("min(50vh, 90%)");
   }  
 
   .mailchimp-container{
-    width: 70vw;
+    width: 80vw;
 
-    margin-top: calc(1vh);
+    margin-top: 1vh;
 
-    height: 10vw;
-    border-radius: 5vw;
+    height: 12vw;
+    border-radius: 6vw;
   }
   .mailchimp-input{
     height: 7vw;
-    font-size: 3.0vw;
+    font-size: 3.25vw;
   }
   .mailchimp-button{
-    height: 8vw;
-    width: 8vw;
+    height: 9vw;
+    width: 9vw;
 
-    border-radius: 4vw;
+    border-radius: 4.5vw;
 
-    margin-right: 1vw;
+    margin-right: 1.5vw;
 
-    background-position: 1vw;
-    background-size: 5vw;
-  }
-  .mailchimp-button:hover{
     background-position: 1.5vw;
+    background-size: 5vw;
   }
-  
   .mobile-banner{
+    display: inline-block;
+  
+    margin-top: 1vh;
+    margin-bottom: 2.5vh;
+
+    border-bottom-style: solid;
+
     font-size: 3.5vw;
+    font-weight: 500;
+    color: $blue;
   }
 }
```

Comment:
needs newline at end of file

Reply:
What does this do?

Reply:
[Long story short](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), convention :p